client/prisma/schema.prisma

```diff
@@ -37,7 +37,8 @@ model Listener {
 }
 
 model Lockup {
-  lockupId  Int @unique
+  id        Int @default(autoincrement()) @id
+  lockupId  Int
```

Comment:
I'd be inclined to have a unique constraint on (user, lockupId) so that we don't ever have duplicates (might happen if for some reason we would run 2 listeners for a brief period of time?)

Reply:
Good thinking added another migration for unique constraint
<img width="759" alt="Screenshot 2022-07-10 at 00 56 33" src="https://user-images.githubusercontent.com/579910/178125165-9651934d-d385-4914-841d-938efe9ea7d8.png">